### DISCUSSION TITLE
New EA Client

### DISCUSSION THREAD

**Discussion by KallanX** [R2Northstar/Northstar#347] [General]
Hey Devs,

With the demise of the origin game launcher and the new EA client being the predominant launcher, does the Northstar mod support the EA client?

If not, do we have plans to support it?

Thanks,
Keith

**Comment by GeckoEidechse** [comment]
Yes, EA app is supported: https://r2northstar.gitbook.io/r2northstar-wiki/installing-northstar/manual-installation
